### PR TITLE
Soulios graphs

### PR DESCRIPTION
**Pull request by soulios**
Preparing new release

### CODE REVIEW

**Comment by soulios** [comment]
I do not know why it doesn't find the directory.When I run the tests locally, there is no issue. :(

**Comment by bernt-matthias** [comment]
Is the directory whose presence is tested [here](https://github.com/yigbt/deepFPlearn/blob/b059e3a94ef8f00aadac7a05151a024290de7991/.github/workflows/pr.yml#L74C18-L74C18) really generated by the call to dfpl. 

You can insert some command line calls like `ls` / `tree` for debugging in the `pr.yaml` file.

File: example/predictgnn.json [hunk 1]
```diff
@@ -1,8 +1,7 @@
 {
   "py/object": "dfpl.options.GnnOptions",
```
Comment (bernt-matthias) [inline example/predictgnn.json:2]: Can you verify that the content of the example files is identical to the state before https://github.com/yigbt/deepFPlearn/pull/22/

Feel free to add additional test data in independent files (maybe even structured in directories). Then add the corresponding test calls to the `pr.yaml` file.

File: .github/workflows/pr.yml [hunk 2]
```diff
@@ -75,11 +75,25 @@ jobs:
           echo "predict result directory missing" >&2 
           exit 1
         fi
-        echo "result lines "$(wc -l example/results_predict/predict_data_AR.csv)
-        if [ "$(cat example/results_predict/predict_data_AR.csv | wc -l)" -lt "2000" ]; then
-          echo "predict result should have at least 2000 lines. But had only $(cat example/results_predict/predict_data_AR.csv | wc -l)" >&2 
+        echo "result lines "$(wc -l example/results_predict/smiles.csv)
+        if [ "$(cat example/results_predict/smiles.csv | wc -l)" -lt "5" ]; then
+          echo "predict result should have at least 5 lines. But had only $(cat example/results_predict/predict_data_AR.csv | wc -l)" >&2
```
Comment (bernt-matthias) [inline .github/workflows/pr.yml:80]: Why did the number of lines change that much? Was this wrong before?

File: example/predictgnn.json (hunk 1 above)
Comment (soulios) [reply example/predictgnn.json:2]: Hooray!all passed!

File: .github/workflows/pr.yml (hunk 2 above)
Comment (soulios) [reply .github/workflows/pr.yml:80]: I did provide another smaller csv to do the preds, but the principle is the same.

File: .github/workflows/pr.yml [hunk 3]
```diff
@@ -75,11 +75,25 @@ jobs:
...
         tree example
 
-        dfpl convert -f example/data
\ No newline at end of file
+        dfpl predictgnn -f example/predictgnn.json
+        if [ ! -d preds_dmpnn/ ]; then
+          echo "predict result directory missing" >&2 
+          exit 1
+        fi
+
```
Comment (bernt-matthias) [inline .github/workflows/pr.yml:97]: Should we implement a simple test here as well, e.g. check the number of lines in the output like above?

File: .github/workflows/pr.yml (hunk 2 above)
Comment (bernt-matthias) [reply .github/workflows/pr.yml:80]: Cool. Good idea.

File: dfpl/autoencoder.py [hunk 4]
```diff
@@ -142,7 +143,7 @@ def train_full_ac(df: pd.DataFrame, opts: options.Options) -> Model:
 
     # If wandb tracking is enabled for autoencoder weights but not for the main program, initialize a new wandb run
     if opts.aeWabTracking and not opts.wabTracking:
-        wandb.init(project=f"AE_{opts.aeSplitType}", config=opts)
+        wandb.init(project=f"AE_{opts.aeSplitType}")
```
[the remark this answers is not in the document]
Comment (soulios) [reply dfpl/autoencoder.py:146]: nope, apparently not. I thought that the parameters wouldnt show up on wandb but they dont need the config option.

**Pull request merged**